stl/inc/xmemory

```diff
@@ -805,6 +805,10 @@ struct _Simple_types { // wraps types from allocators with simple addressing for
 _INLINE_VAR constexpr size_t _Asan_granularity      = 8;
 _INLINE_VAR constexpr size_t _Asan_granularity_mask = _Asan_granularity - 1;
 
+// Represents whether ASan container annotations are enabled a given allocator.
+template <class>
+constexpr bool _Is_ASan_enabled_for_allocator = true;

```

Comment:
Nice!

I may have missed some conversation about it, but I'm wondering if the name should be changed. The feature is disabling annotations based on the _container's_ underlying allocator. 

Since the options for compiler are `_DISABLE_X_ANNOTATION` should we try to follow suite with `Disable` in the name?

Reply:
>Since the options for compiler are _DISABLE_X_ANNOTATION should we try to follow suite with Disable in the name?

Oh I didn't realize the compiler options / flags followed that naming convention? Do you have a link or example where I can see this?

But yeah, don't feel strongly about naming. If it matches the flags, I'm happy to have `Disabled` over `Enabled` in the name. I'll tackle that on my next push.

Reply:
Yeah, I'm basically commenting on staying in line with [these](https://github.com/microsoft/STL/blob/9b38fb9ccca90fd8b31fd37d4512b190aeef4fe7/stl/inc/__msvc_sanitizer_annotate_container.hpp#L18-L27)

Reply:
Incorporated: https://github.com/microsoft/STL/pull/5241/commits/57e7affd7f3f5d6f32c255c0a7b822c7006f7551